src/styles/DenHaagBrand/src/denhaag-brand.module.css

```diff
@@ -1,5 +1,10 @@
 :root {
-  /* Colors */
+  font-size: 16px;
```

Comment:
@bddjong I recall a discussion we had about avoiding `px` values... was an hardcoded `16px` the conclusion? Would love to hear more about this.

Reply:
I actually kept the font-size as a hardcoded px because I was under the impression that it would remove ambiguity about the font size that all other rem values are based on. In addition, I don't believe the base font size is necessarily identical in all browsers, so setting it in the Den Haag brand package would enforce the desired scale. 

If this is bad practice I see no problem with removing it!

Reply:
I think allowing ambiguity of the base font size is actually a goal in itself. The folks who change the default font size of their browsers, are likely to need it to be able to read your texts! Why else would they have changed it?

More info: [Pixels vs. Relative Units in CSS: why it’s still a big deal](https://www.24a11y.com/2019/pixels-vs-relative-units-in-css-why-its-still-a-big-deal/)

Reply:
Fixed